test/long/org/apache/cassandra/io/sstable/CQLSSTableWriterLongTest.java

```diff
@@ -76,7 +76,7 @@ public void testWideRow() throws Exception
                                                   .inDirectory(dataDir)
                                                   .forTable(schema)
                                                   .using(insert)
-                                                  .withBufferSizeInMiB(1)
+                                                  .withBufferSizeInMB(1)
```

Comment:
Intentionally changed to the old method In tests so we test the whole path until the deprecated ones is removed